test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -1288,6 +1288,34 @@ bool VerifyRemoveAllAsync()
     return true;
 }
 
+
+bool VerifyForegroundRegistrationSuccess()
+{
+    winrt::AppNotificationManager::Default().Unregister();
+    try
+    {
+        winrt::event_token token{ winrt::AppNotificationManager::Default().NotificationInvoked([](const auto&, winrt::AppNotificationActivatedEventArgs const& toastArgs) {}) };
+    }
+    catch (...)
+    {
+        return false;
+    }
+    return true;
+}
+
+bool VerifyForegroundRegistrationFails()
```

Comment:
Please either write a comment specifying why this is expected to fail or provide a more descriptive test name.

Reply:
+1